library/src/main/scala/scala/scalajs/js/Array.scala

```diff
@@ -183,4 +183,10 @@ object Array extends Object {
 
   /** Returns true if the given value is an array. */
   def isArray(arg: Any): Boolean = native
+
+  /** Creates a new array from the given iterable */
+  def from[A](iterable: Iterable[A]): Array[A] = native
+
+  /** Creates a new array from the given iterator */
+  def from[A](iterator: Iterator[A]): Array[A] = native
```

Comment:
Array.from doesn't accept iterator. It will cause TypeError:

![image](https://user-images.githubusercontent.com/705555/75313893-95893e80-58a1-11ea-8d85-dbfd4415ba73.png)